docs/guide/creating-custom-rules.md

```diff
@@ -21,7 +21,7 @@ Let's try to create a rule for checking that a value is a valid [RGB color].
 
 ### Creating a rule
 
-The 1st step is creation a rule:
```

Comment:
In my variation "of" is missing, but it's usable too.

Reply:
Step is an action. "Creation" is a noun. I'd expect a verb.